scassandra/src/main/scala/com/evolutiongaming/scassandra/syntax.scala

```diff
@@ -89,23 +96,56 @@ object syntax {
     }
   }
 
-
-  implicit class ScassandraGettableByIdxDataOps(val self: GettableByIndexData) extends AnyVal {
+  implicit class ScassandraGettableByIdxDataOps(val self: GettableByIndexData)
+      extends AnyVal {
 
     def decodeAt[A](idx: Int)(implicit decode: DecodeByIdx[A]): A = {
       decode(self, idx)
     }
   }
 
-
   implicit def toCqlOps[A](a: A): ToCql.Ops.IdOps[A] = new ToCql.Ops.IdOps(a)
 
-
   implicit class ScassandraStatementOps(val self: Statement) extends AnyVal {
 
     def trace(enable: Boolean): Statement = {
       if (enable) self.enableTracing()
       else self.disableTracing()
     }
   }
+
+  implicit class ScassandraCodexPutSyntax[D <: GettableData & SettableData[D]](
```

Comment:
why `CodeX`?

Reply:
Fixed and then deleted in favor of `UpdateXXX`